### PR TITLE
intial support for multiple databases

### PR DESCRIPTION
**Pull request by fdellwing**
Relates to #26 

- [x] Support multiple databases for dumping
- [x] ~~Support multiple databases for streaming~~ Fix streaming (works as before)
- [x] Normalize log output in loader
- [x] Tests

### CODE REVIEW

**Comment by BohuTANG** [comment]
Cool, Thanks for the contribution.
But we need some unit-tests to make the CI happy.

**Comment by fdellwing** [comment]
It's on the agenda. Will work on streamer next, it is broken atm.

**Comment by fdellwing** [comment]
@BohuTANG Should streamer support more than one database?

**Comment by BohuTANG** [comment]
> @BohuTANG Should streamer support more than one database?

No need, streamer maybe discared someday.

**Comment by fdellwing** [comment]
I'm not understanding your go-mysqlstack enough to understand why this test fails. It also seems like, you mock-server is not able to work with multiple databases?

File: src/common/dumper_test.go [hunk 1]
```diff
@@ -142,3 +142,123 @@ func TestDumper(t *testing.T) {
...
+				sqltypes.MakeTrusted(querypb.Type_VARCHAR, []byte("t1-05-11")),
+			},
+			{
+				sqltypes.MakeTrusted(querypb.Type_VARCHAR, []byte("t2-05-11")),
+			},
+		}}
+
+	// fakedbs.
+	{
+		fakedbs.AddQueryPattern("use .*", &sqltypes.Result{})
```
Comment (BohuTANG) [inline src/common/dumper_test.go]: Test errors:
 2020/03/16 12:17:09.147700 server.go:240:    	 [ERROR] 	server.handle.query.from.session[1].error:mock.handler.query[show databases].error[can.not.found.the.cond.please.set.first].query[SHOW DATABASES]

Here, we should add:
```
fakedbs.AddQueryPattern("show databases", &sqltypes.Result{?...})
```

Comment (fdellwing) [reply src/common/dumper_test.go]: `{?...}` gives a build error and `{}` seems to work, but it seems like the result of `SHOW DATABASES` is empty as he dumps nothing => `dumping.all.done.cost[0.00sec].allrows[0].allbytes[0].rate[0.00MB/s]`

Comment (BohuTANG) [reply src/common/dumper_test.go]: Hmm, you should fill the values replace the ?... to mock the 'show databases' result.
&sqltypes.Result{} is empty.
Here is an example to mock `show tables from xx` result:
https://github.com/xelabs/go-mydumper/blob/master/src/common/dumper_test.go#L92

Comment (fdellwing) [reply src/common/dumper_test.go]: Ok, I thought I got it. But I seem to have a minor problem with
```
fakedbs.AddQueryPattern("select * from `test1`.*", selectResult1)
```

**Comment by BohuTANG** [comment]
Please run 'make test' first and make it all passed before the push, don't try the github CI, it's waste time.

**Comment by BohuTANG** [comment]
Cool, Thanks for the contribution, I will review it tomorrow, good night.

**Pull request merged**